mino/minogrpc/session/addr.go

```diff
@@ -139,11 +137,26 @@ func (a Address) MarshalText() ([]byte, error) {
 // String implements fmt.Stringer. It returns a string representation of the
 // address.
 func (a Address) String() string {
+	url := "grpcs://"
+	switch a.connectionType {
+	case mino.ACTgRPCS:
+		url += a.host
+	case mino.ACTgRPC:
+		url = "grpc://" + a.host
+	case mino.ACThttps:
+		url = "https://"
+		hp := strings.Split(a.host, ":")
+		if len(hp) <= 1 {
+			url += a.host
+		} else {
+			url += hp[1]
+		}
+	}
```

Comment:
In python, I would do this using a "dispatch table", but it doesn't look good in golang 
https://chat.openai.com/share/e24fcf2d-a14b-4198-83e8-2910e17c7cd9

Reply:
In python, I would use a lambda function.. ChatGPT didn't use lambdas, so I guess go lang doesn't have them?

Reply:
You can have lambda functions in go. However I don't like making the code more intelligent than it needs to be.

What would be the advantage of a function table compared to a `switch`/`case` construction?